config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -2522,23 +2524,25 @@ presubmits:
     trigger: (?m)^/test( | .* )pull-security-kubernetes-verify,?($|\s.*)
   - agent: kubernetes
     always_run: false
+    branches:
+    - master
     cluster: security
-    context: pull-security-kubernetes-verify-podutil
+    context: pull-security-kubernetes-verify
     decorate: true
     labels:
       preset-dind-enabled: "true"
       preset-service-account: "true"
-    name: pull-security-kubernetes-verify-podutil
+    name: pull-security-kubernetes-verify
     path_alias: k8s.io/kubernetes
-    rerun_command: /test pull-security-kubernetes-verify-podutil
+    rerun_command: /test pull-security-kubernetes-verify
     spec:
       containers:
       - args:
         - --ssh=/etc/ssh-security/ssh-security
         - ./hack/jenkins/verify-dockerized.sh
         command:
         - runner.sh
-        image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20190110-c23317e88
```

Comment:
shouldn't these have a `-master`?

Reply:
gooooooood catch :joy:

Reply:
done, yeah I copied this blindly from the other job